wheels/build/wheels.yml

```diff
@@ -62,6 +62,20 @@ packages:
         prebuild:
             wheel: for f in sqlite3.h sqlite3.c sqlite3ext.h; do cp ${SRC_ROOT_1}/$f $f || return 1; done && echo 'include_dirs=.' >>${SRC_ROOT_0}/setup.cfg
         build_args: build_static bdist_wheel
+    python-ldap:
+        version: 2.4.27
+        src:
+            - http://mirror.eu.oneandone.net/software/openldap/openldap-release/openldap-2.4.44.tgz
+            - https://pypi.python.org/packages/fc/99/9eed836fe4d916792994838df125da9c25c5f7c31abfbf6f0ab076e5f419/python-ldap-2.4.27.tar.gz
```

Comment:
That's what I thought at the beginning, but pysqlite is broken (but not used looking at the comment) ;)

In the code, just before running setup.py, it [cd to a "root" directory](https://github.com/galaxyproject/starforge/blob/master/starforge/forge/wheels.py#L202) which is defined [here](https://github.com/galaxyproject/starforge/blob/master/starforge/forge/wheels.py#L134) while looping on the src lines (and pip path is the first one). So root directory ends up being the last one from the src list.

Maybe it should be changed in the code, but I fear it could break other packages